src/app/page.js

```diff
@@ -62,26 +63,26 @@ export default async function HelpsPage() {
           <Grid
             container
             style={{
-              display: "flex",
-              width: "100%",
-              justifyContent: "center",
+              display: 'flex',
+              width: '100%',
+              justifyContent: 'center',
               marginTop: 50,
             }}
           ></Grid>
           <Grid container>
             <Grid
               style={{
-                display: "flex",
-                flexDirection: "row",
+                display: 'flex',
+                flexDirection: 'row',
                 marginTop: 20,
-                alignItems: "center",
+                alignItems: 'center',
               }}
             ></Grid>
           </Grid>
           <Grid
             container
             style={{
-              justifyContent: "center",
+              justifyContent: 'center',
```

Comment:
please make sure to push only the changes

Reply:
@suzuya1331, could you please rebase onto main? We've added a prettier config file which should help prevent this issue.